src/msg/async/Stack.h

```diff
@@ -32,6 +32,7 @@ class ConnectedSocketImpl {
   virtual void shutdown() = 0;
   virtual void close() = 0;
   virtual int fd() const = 0;
+  virtual int set_priority(int sd, int prio, int domain) = 0;
```

Comment:
Again, given that `NetHandler::set_priority()` swallows all errors, what is the purpose of returning an int here?

Reply:
change to not swallows all errors, thanks